episodes/07-thresholding.md

```diff
@@ -348,13 +348,17 @@ plt.xlim(0, 1.0)
 ![](fig/maize-root-cluster-histogram.png){alt='Grayscale histogram of the maize root image'}
 
 The histogram has a significant peak around 0.2, and a second,
-smaller peak very near 1.0.
+shallower peak near 0.6.
 Thus, this image is a good candidate for thresholding with Otsu's method.
 The mathematical details of how this works are complicated (see
 [the scikit-image documentation](https://scikit-image.org/docs/dev/api/skimage.filters.html#threshold-otsu)
 if you are interested),
 but the outcome is that Otsu's method finds a threshold value between
 the two peaks of a grayscale histogram.
```

Comment:
```suggestion
but the outcome is that Otsu's method finds a threshold value that separates the peaks of a grayscale histogram that might correspond to background and foreground pixel values.
```

Reply:
I think that "a value between the peaks" is correct and that "separating" applies to "the foreground from the background" or vice-versa. So, I would suggest something along these lines:

> Otsu's method finds a threshold value between the two peaks of a grayscale histogram: This threshold thus separates higher from lower pixel values, which might correspond to the foreground and background respectively.

Reply:
```suggestion
but the outcome is that Otsu's method finds a threshold value between
the two peaks of a grayscale histogram.
```
but the outcome is that Otsu's method finds a threshold value between the two peaks of a grayscale histogram which might correspond well to the foreground and background depending the data and application.